ofrak_core/test_ofrak/unit/test_ofrak_context.py

```diff
@@ -14,6 +14,32 @@
 
 from ofrak.core.filesystem import FilesystemRoot
 
+from ofrak.model.component_model import ComponentExternalTool
+from ofrak.core.squashfs import _UnsquashfsV45Tool
+from ofrak.core.strings_analyzer import _StringsToolDependency
+
+
+def test_path_auth_error(monkeypatch, caplog):
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+
+    if os.name != "posix":
+        LOGGER.warning("test_path_auth_error is currently only supported by POSIX systems.")
+        return
+
+    monkeypatch.setenv("PATH", "/root")
+
+    for tool in (
+        ComponentExternalTool("apktool", "", "-version"),
+        _UnsquashfsV45Tool(),
+        _StringsToolDependency(),
+    ):
+        task = loop.create_task(tool.is_tool_installed())
+        loop.run_until_complete(task)
```

Comment:
Why do it this way instead of just making the test `async` and using [`pytest.mark.asyncio`](https://pytest-asyncio.readthedocs.io/en/latest/)?